recipes-core/images/iot2050-image-swu-example.bb

```diff
@@ -13,11 +13,11 @@ require recipes-core/images/swupdate.inc
 
 inherit image_uuid
 
-IMAGE_TYPEDEP_wic = "squashfs"
-IMAGE_TYPEDEP_wic_secureboot = "verity"
+IMAGE_TYPEDEP:wic = "squashfs"
+IMAGE_TYPEDEP:wic:secureboot = "verity"
```

Comment:
This looks strange... should this be `IMAGE_TYPEDEP:secureboot`?

Reply:
It's actually a conditional: If we build a non-secure image, we use plain squashfs. If secure is on, that `secureboot` override is set, and we wrap dm-verity around the squashfs. But all that is prior to Felix' recent isar-cip-core image type changes. Those are my major todo, and maybe they will change this as well.